_data/cezarmathe/cezarmathe.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Armand Cezar Mathe
+institution: Andrei Muresanu National College, Dej, Romania
+profile_pic: cezarmathe.jpg
```

Comment:
```suggestion
profile_pic: cezarmathe.png
```

Reply:
Do I have to make more changes?
I changed the image extension to .jpg because it is a jpg image, it's not a png.